Yoast/wordpress-seo
# Twitter site

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Test instructions
- Set the site type to company and fill in the twitter handle under social.
- Go to a page and see `twitter:site` filled with a twitter handle.
- Set the site type to person and under your profile fill in the twitter handle. A different one to see the difference.
- Go to a page and see `twitter:site` filled in with a twitter handle.
